src/main/resources/enchantCraftingList.json

```diff
@@ -0,0 +1,169 @@
+{
+  "Normal": {
+    "ENCHANTED_PUMPKIN":{"Material":"PUMPKIN"},
+    "ENCHANTED_POTATO": {"Material":"POTATO_ITEM"},
+    "ENCHANTED_SLIME_BALL": { "Material": "SLIME_BALL"},
+    "ENCHANTED_RED_MUSHROOM": { "Material":"RED_MUSHROOM"},
+    "ENCHANTED_CARROT": { "Material":"CARROT_ITEM"},
+    "ENCHANTED_OAK_LOG": { "Material":"LOG"},
+    "ENCHANTED_BIRCH_LOG": { "Material":"LOG:2"},
+    "ENCHANTED_GUNPOWDER": { "Material":"SULPHUR"},
+    "ENCHANTED_MELON": { "Material":"MELON"},
+    "ENCHANTED_SUGAR": { "Material":"SUGAR_CANE"},
+    "ENCHANTED_BLAZE_ROD": { "Material":"ENCHANTED_BLAZE_POWDER"},
+    "ENCHANTED_DIAMOND": { "Material": "DIAMOND"},
+    "ENCHANTED_BROWN_MUSHROOM": { "Material": "BROWN_MUSHROOM"},
+    "ENCHANTED_RAW_SALMON": { "Material": "RAW_FISH:1"},
+    "ENCHANTED_COOKED_MUTTON": { "Material":"ENCHANTED_MUTTON"},
+    "ENCHANTED_EMERALD": { "Material":"EMERALD"},
+    "ENCHANTED_SPIDER_EYE": { "Material":"SPIDER_EYE"},
+    "ENCHANTED_MELON_BLOCK": { "Material": "ENCHANTED_MELON"},
+    "ENCHANTED_PUFFERFISH": { "Material":"RAW_FISH:3"},
+    "ENCHANTED_NETHERRACK": { "Material":"NETHERRACK"},
+   "MUTANT_NETHER_STALK": {"Material":  "ENCHANTED_NETHER_STALK"},
+    "ENCHANTED_BAKED_POTATO": { "Material": "ENCHANTED_POTATO"},
+    "ENCHANTED_COBBLESTONE": { "Material":"COBBLESTONE"},
+   "ENCHANTED_BLAZE_POWDER": { "Material": "BLAZE_ROD"},
+   "ENCHANTED_SNOW_BLOCK": { "Material": "SNOW_BLOCK"},
+    "ENCHANTED_BREAD": { "Material": "WHEAT"},
+   "ENCHANTED_LAPIS_LAZULI_BLOCK": { "Material": "ENCHANTED_LAPIS_LAZULI"},

```

Comment:
I don't know which other items this applies to, but the data doesn't show in /bn find

Reply:
no real order